experimental/README.md

```diff
@@ -25,15 +25,15 @@ Experimental samples are organized per platform.
 
 | Sample Name           | Description                                                                    | .NET CORE   | NodeJS      | .NET Web API | Typescript  |
 |-----------------------|--------------------------------------------------------------------------------|-------------|-------------|--------------|-------------|
-|bot translation library| The sample shows how to use the library through Middleware to support multilingual interaction with bots in general and LUIS bots in particular.                                                                                                 |[View][cs#1] |  |  |  |
+|bot translation library| The sample shows how to use the library through Middleware to support multilingual interaction with bots in general and LUIS bots in particular.                                                                                                 |[View][cs#1] | [View][js#1] |  | [View][ts#1] |
 
 
 [cs#1]:csharp_dotnetcore/MultilingualLuisBot
 
 [wa#2]:csharp_webapi/#
 
-[ts#1]:javascript_typescript/#
+[ts#1]:javascript_typescript/60.multilingual-luis-bot
 
-[js#1]:samples/javascript_nodejs/#
+[js#1]:javascript_typescript/60.multilingual-luis-bot
```

Comment:
Why is the Node.js link pointing towards the TypeScript sample? If we're going to have a corresponding Node.js sample we should wait until that sample is done and then update the readme. If there aren't plans to create a Node.js sample, we should leave it blank. 

@sgellock, @daveta or @cleemullins, could one of you chime in? What's the plan here?

Reply:
I think this is a bug I introduced.  I can fix it up

Reply:
@stevengum The NodeJS link was removed. The original link template was left in place for when the  NodeJS same is ready.